data/channels/c0nc0rdance.yml

```diff
@@ -7,8 +7,17 @@ providers:
     name: C0nc0rdance
     slug: C0nc0rdance
     url: https://youtube.com/C0nc0rdance
-    description: ""
-    subscribers: 54153
+    description: |-
```

Comment:
scuffed as symbols
also
For consistency should this content be surrounded by quotation marks?